lookout/style/format/analyzer.py

```diff
@@ -284,7 +282,7 @@ def _generate_comments_data(
             feature_extractor_output, bblfsh_stub: "bblfsh.aliases.ProtocolServiceStub",
             rules: Rules) -> Iterable[FixData]:
         X, y, (vnodes_y, vnodes, vnode_parents, node_parents) = feature_extractor_output
-        y_pred, rule_winners, new_rules = rules.predict(X=X, vnodes_y=vnodes_y, vnodes=vnodes,
+        y_pred, rule_winners, new_rules = rules.predict(X=X, y=y, vnodes_y=vnodes_y, vnodes=vnodes,
```

Comment:
Please rename it to `y_fallback`

Reply:
as mentioned during our quick call this morning with @zurk, we should not use `y` during the predict phase since we are trying to remove the dependency on `y` in most of the codebase to be able to predict on unseen targets